web/src/scheduled_messages.js

```diff
@@ -15,34 +15,71 @@ import * as narrow from "./narrow";
 import * as people from "./people";
 import * as popover_menus from "./popover_menus";
 import * as stream_data from "./stream_data";
+import * as timerender from "./timerender";
 
 export let scheduled_messages_data = [];
 
+const st = compute_later_send_times();
```

Comment:
I think we really don't want to call this only once on startup -- if the day changes with the app still open, the values will all end up wrong.

Reply:
Really good point. I hadn't thought of that.

Fixing this might actually fix my larger issue of user-preferred 24-hour times not being reflected in the modal, too.